tests/new_unit_tests/Commands/ArtCommandTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Pantheon\Terminus\UnitTests\Commands;
+
+use Pantheon\Terminus\Tests\CommandTestCase;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class ArtCommandTest extends CommandTestCase
+{
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->setInput(['command' => 'art', 'name' => 'hello']);
+    }
+
+    public function testArtPrintsArt()
+    {
+        $this->assertEquals('Hello World!', $this->runCommand()->getOutput());
```

Comment:
I agree, I actually made an assumption in `getOutput()` that probably won't hold true in many cases because it fit my minimalist assumption. I wanted to git'r done and see if the general consensus is that it was useful before refining too far though. This was inspired by Laravel's testing classes